src/parser/DefaultParser.js

```diff
@@ -14,6 +14,15 @@ class DefaultParser {
       links: inputs,
     };
   }
+
+  /**
+   * Indicate if this parser can parse this file.
+   * @param {String} [fileName] - Name of file.
+   * @return {Boolean} Boolean of if this file can be parse or not.
```

Comment:
I think the wording of this comment is weird, but I'm not sure.

Maybe it's better "Boolean that indicates if this file can be parsed or not".

Reply:
Yes it's better, I push the changes :)